rnd/autogpt_server/autogpt_server/util/settings.py

```diff
@@ -74,6 +74,7 @@ class Secrets(UpdateTrackingModel["Secrets"], BaseSettings):
     openai_api_key: str = Field(default="", description="OpenAI API key")
     anthropic_api_key: str = Field(default="", description="Anthropic API key")
     groq_api_key: str = Field(default="", description="Groq API key")
+    ollama_api_key: str = Field(default="", description="Ollama API key")
```

Comment:
this is not needed, but the validation complains on this. Alternatively i can create a "LocalLLMCall" block

Reply:
https://github.com/Significant-Gravitas/AutoGPT/pull/7505/files#r1684171354